example/constants/styles.ts

```diff
@@ -0,0 +1,105 @@
+import { StyleSheet } from 'react-native'
+import { Colors } from './colors'
+
+export const GlobalStyles = StyleSheet.create({
+  screenContainer: {
+    flex: 1,
+    backgroundColor: Colors.swmBlue,
+  },
+  centeredContainer: {
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+    padding: 20,
+    backgroundColor: '#f5f5f5',
+  },
+  contentContainer: {
+    padding: 20,
+  },
+
+  card: {
+    backgroundColor: '#fff',
+    borderRadius: 10,
+    padding: 15,
+    marginBottom: 15,
+    shadowColor: '#000',
+    shadowOffset: { width: 0, height: 2 },
+    shadowOpacity: 0.1,
+    shadowRadius: 4,
+    elevation: 3,
+  },
+  infoBox: {
+    backgroundColor: '#f8f8f8',
+    padding: 10,
+    borderRadius: 5,
```

Comment:
It’s common practice to use only multiples of 4. For example: [Layout Basics: Grid Systems and the 4px Grid](https://blog.designary.com/p/layout-basics-grid-systems-and-the-4px-grid)

```suggestion
    borderRadius: 4,
```